packages/server/package.json

```diff
@@ -126,6 +127,7 @@
     "pouchdb-all-dbs": "1.0.2",
     "pouchdb-find": "^7.2.2",
     "pouchdb-replication-stream": "1.2.9",
+    "redis": "4",
```

Comment:
in backend-core we use `io-redis` but here we need `redis`. I don't see any usages of the `redis` dependency here, how come it's needed?

Reply:
Its required for the koa2 redis store, it is based on the Redis client rather than the IORedis client that we use. Weirdly the `koa2-ratelimit` library doesn't include its own store dependencies.